File: src/Squidex/app/features/content/pages/messages.ts
/*
 * Squidex Headless CMS
 *
 * @license
 * Copyright (c) Squidex UG (haftungsbeschränkt). All rights reserved.
 */

import { Version } from '@app/shared';

export class ContentVersionSelected {
    constructor(
        public readonly version: Version
    ) {
    }
}

File: backend/src/Squidex.Infrastructure/EventSourcing/Grains/Messages.cs
// ==========================================================================
//  Squidex Headless CMS
// ==========================================================================
//  Copyright (c) Squidex UG (haftungsbeschraenkt)
//  All rights reserved. Licensed under the MIT license.
// ==========================================================================

#pragma warning disable MA0048 // File name must match type name
#pragma warning disable SA1313 // Parameter names should begin with lower-case letter

namespace Squidex.Infrastructure.EventSourcing.Grains
{
    public sealed record EventConsumerReset(string EventConsumer)
    {
    }

    public sealed record EventConsumerStart(string EventConsumer)
    {
    }

    public sealed record EventConsumerStop(string EventConsumer)
    {
    }
}
